app/serializers/favorite_serializer.rb

```diff
@@ -0,0 +1,4 @@
+class FavoriteSerializer
+  include FastJsonapi::ObjectSerializer
+  attributes :breed
```

Comment:
I originally had this as `belongs_to` but the serialized JSON wasn't what seemed to be expected. This pleasantly returns all the breed information!